app.py

```diff
@@ -11,8 +11,7 @@
 load_dotenv()
 
 import jwt
-from datetime import datetime
-
+import datetime
```

Comment:
why  you decide to use `datetime` instead of `from datetime import datetime`

Reply:
Sir both are correct but previously this module gives an error so that's why I tried to change it.